aiozk/test/test_client.py

```diff
@@ -45,6 +45,22 @@
     await asyncio.wait_for(zk.session.close(), 2)
 
 
+
+@pytest.mark.asyncio
+async def test_inconsistent_zxid():
+    async def coro():
+        zk = get_client()
+        await zk.start()
+        # simulate failed connection
+        await zk.session.close()
+        zk.session.last_zxid = 1231231241312312
```

Comment:
If I comment out this string and run the test on codebase without any fixes - it doesn't work.
Probably we need a test that doesn't work if we override zxid and works if we don't override it

Reply:
it should fail with timeout exception, I will check it

Reply:
I mean code without this line should pass the test.
```python
# zk.session.last_zxid = 1231231241312312
```

Reply:
On codebase without fixes it will not work even if we comment `zk.session.last_zxid`. We can't reconnect to zk because of session holds in closing state. I will add test which checks only session reconnect